iceprod/server/data/www_templates/dataset_detail.html

```diff
@@ -31,10 +31,10 @@ <h2>Settings</h2>
 
 {% if (current_user) %}
 <div class="action">
-    <button onclick='call("reset_dataset");'>Reset</button>
-    <button onclick='call("hard_reset_dataset");'>Hard Reset</button>
-    <button onclick='call("suspend_dataset");'>Suspend</button>
-    <button onclick='call("truncate_dataset");'>Truncate</button>
+    <button onclick='set_dataset_status("{{ dataset_id }}", "reset", "{{ passkey }}", filter=["&task_status=failed", "&task_status=suspended"]);'>Reset</button>

```

Comment:
I'm unsure that the filter will work the way you think it will.  But we can leave this until we try it.